src/microbe_stage/MicrobeStage.cs

```diff
@@ -208,10 +214,19 @@ public override void _Process(float delta)
         microbeAISystem.Process(delta);
         microbeSystem.Process(delta);
 
-        // TODO: only update these like 60 times a second
-        if (GameWorld.Map.CurrentPatch != null)
-            patchManager.UpdatePatchBiome(GameWorld.Map.CurrentPatch);
-        patchManager.UpdateAllPatchLightLevels();
+        elapsedSinceLightLevelUpdate += delta;
+        if (elapsedSinceLightLevelUpdate > Constants.LIGHT_LEVEL_UPDATE_INTERVAL)
+        {
+            elapsedSinceLightLevelUpdate = 0;
+
+            if (GameWorld.Map.CurrentPatch != null)
+            {
+                patchManager.UpdatePatchBiome(GameWorld.Map.CurrentPatch);
+                patchManager.UpdateAllPatchLightLevels();
```

Comment:
Why is here a change of logic?

Reply:
These were all marked as needing to be updated less frequently and, as far as I can tell, all require an existing `CurrentPatch`. The order of execution is the same as before.

Reply:
Okay. Just to make sure because it seems that UpdateAllPatchLightLevels() will be called even there isn't a current patch.